libcxx/test/std/thread/thread.threads/thread.thread.class/thread.thread.constr/constr.compile.fail.cpp

```diff
@@ -9,12 +9,11 @@
 // <thread>
 
 // class thread
-//     template <class _Fp, class ..._Args,
-//         explicit thread(_Fp&& __f, _Args&&... __args);
+//     template <class _Func, class ..._Args,
+//         explicit thread(_Func&& __f, _Args&&... __args);
```

Comment:
Actually here we should use the non-ugly names as used in the Standard's synopsis.

Reply:
+1 -- it's a preexisting issue, but I think we previously agreed that when copying from synopsis, we should just do a direct copy-paste without adjusting any names or formatting. I'm ok with the change (IMO fixing it is out of scope for this patch) or with simply not touching this file.